spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -40,7 +40,7 @@
                               to_text_string)
 from spyder.utils import icon_manager as ima
 from spyder.utils.qthelpers import add_actions, create_action, keybinding
-
+from .basedialog import BaseDialog

```

Comment:
@jsh9 I think this import should be something like `spyder.plugins.variableexplorer.widgets.basedialog import BaseDialog` (and should be changed also for the other dialogs)

Reply:
Fixed. Thank you.